packages/yuudachi/src/functions/reports/createReport.ts

```diff
@@ -0,0 +1,79 @@
+import type { Message, Snowflake } from 'discord.js';
+import type { Sql } from 'postgres';
+import { container } from 'tsyringe';
+import type { CamelCasedProperties } from 'type-fest';
+import { type RawReport, transformReport } from './transformReport.js';
+import { kSQL } from '../../tokens.js';
+import type { PartialAndUndefinedOnNull } from '../../util/types.js';
+
+export type Report = PartialAndUndefinedOnNull<CamelCasedProperties<RawReport>> & {
+	type: ReportType;
+	status: ReportStatus;
+};
+export interface CreateReport {
+	guildId: Snowflake;
+	reportId?: number;
+	type: ReportType;
+	status?: ReportStatus;
+	message?: Message;
+	targetId: Snowflake;
+	targetTag: string;
+	authorId: Snowflake;
+	authorTag: string;
+	reason: string;
+	attachmentUrl?: string | undefined | null;
+	logMessageId?: Snowflake | undefined | null;
+	refId?: number | undefined | null;
+}
```

Comment:
Any reason `CreateReport` isn't using `Report` and omitting the ones that need different values?

Reply:
Saying that I forgot to change after the merge is a good excuse? 😅

Reply:
Fixed in 33743ae